tests/unit/spanner_dbapi/test_cursor.py

```diff
@@ -862,3 +862,31 @@ def test_fetchmany_retry_aborted_statements_checksums_mismatch(self):
                     cursor.fetchmany(len(row))
 
                 run_mock.assert_called_with(statement, retried=True)
+
+    def test_ddls_with_semicolon(self):
+        """
+        Check that one script with several DDL statements separated
+        with semicolons is splitted into several DDLs.
+        """
+        from google.cloud.spanner_dbapi.connection import connect
+
+        EXP_DDLS = [
+            "CREATE TABLE table_name (row_id INT64) PRIMARY KEY ()",
+            "DROP TABLE table_name",
+        ]
+
+        with mock.patch(
+            "google.cloud.spanner_v1.instance.Instance.exists", return_value=True,
+        ):
+            with mock.patch(
+                "google.cloud.spanner_v1.database.Database.exists", return_value=True,
+            ):
+                connection = connect("test-instance", "test-database")
+
+        cursor = connection.cursor()
+        cursor.execute(
+            "CREATE TABLE table_name (row_id INT64) PRIMARY KEY ();"
+            "DROP TABLE table_name;"
```

Comment:
Set semicolons in both lines to check that no empty DDLs are executed (as `split()` can produce empty lines, if the separator is at the end of the line)